frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/components/ComponentPortletUtils.java

```diff
@@ -148,6 +149,12 @@ public static void updateVendorFromRequest(PortletRequest request, Vendor vendor
         setFieldValue(request, vendor, Vendor._Fields.URL);
     }
 
+    public static void updateTodoFromRequest(PortletRequest request, Todo todo) {
```

Comment:
Is this class the correct place for this method? I can't see on the first impression that it has something to do with components.

Reply:
This sounds correct. I'll check for a better place.

Reply:
I introduced TodosPortletUtil because there was no appropriate place.